### ISSUE TITLE
Static ENI IP eth1 - unable to send state change

### ISSUE THREAD

**Issue by FLavalliere** [aws/amazon-ecs-agent#1683]
### Summary
We want to be able to provision some ECS hosts with some static IP Addresses. Few reasons, some solutions requires Static IP Address / Mac Address.

### Description
We currently provision an ASG and remove eth0 and add the static as eth1 to the VM before launching the ECS Agent.

### Expected Behavior
Everything seems to be functional (at first sight).

### Observed Behavior

We see the following error repeatively in the log.

2018-11-08T20:18:24Z [WARN] Udev watcher reconciliation: unable to send state change: udev watcher send ENI state change: eni not managed by ecs: 0a:20:e9:96:8d:92

### Environment Details
ifconfig

```
[root@ip-192-168-12-37 ~]# ifconfig
docker0   Link encap:Ethernet  HWaddr 02:42:C5:99:3A:D0
          inet addr:172.17.0.1  Bcast:172.17.255.255  Mask:255.255.0.0
          inet6 addr: fe80::42:c5ff:fe99:3ad0/64 Scope:Link
          UP BROADCAST MULTICAST  MTU:1500  Metric:1
          RX packets:15 errors:0 dropped:0 overruns:0 frame:0
          TX packets:25 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:0
          RX bytes:1009 (1009.0 b)  TX bytes:2357 (2.3 KiB)

eth1      Link encap:Ethernet  HWaddr 0A:20:E9:96:8D:92
          inet addr:192.168.12.24  Bcast:192.168.12.255  Mask:255.255.255.0
          inet6 addr: fe80::820:e9ff:fe96:8d92/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:9001  Metric:1
          RX packets:92679 errors:0 dropped:0 overruns:0 frame:0
          TX packets:119548 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000
          RX bytes:15782754 (15.0 MiB)  TX bytes:15638404 (14.9 MiB)

lo        Link encap:Local Loopback
          inet addr:127.0.0.1  Mask:255.0.0.0
          inet6 addr: ::1/128 Scope:Host
          UP LOOPBACK RUNNING  MTU:65536  Metric:1
          RX packets:6480 errors:0 dropped:0 overruns:0 frame:0
          TX packets:6480 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000
          RX bytes:507489 (495.5 KiB)  TX bytes:507489 (495.5 KiB)
```

### Supporting Log Snippets
```
018-11-08T20:23:24Z [WARN] Udev watcher reconciliation: unable to send state change: udev watcher send ENI state change: eni not managed by ecs: 0a:20:e9:96:8d:92
2018-11-08T20:23:54Z [WARN] Udev watcher reconciliation: unable to send state change: udev watcher send ENI state change: eni not managed by ecs: 0a:20:e9:96:8d:92
2018-11-08T20:24:24Z [WARN] Udev watcher reconciliation: unable to send state change: udev watcher send ENI state change: eni not managed by ecs: 0a:20:e9:96:8d:92
2018-11-08T20:24:54Z [WARN] Udev watcher reconciliation: unable to send state change: udev watcher send ENI state change: eni not managed by ecs: 0a:20:e9:96:8d:92
2018-11-08T20:25:24Z [WARN] Udev watcher reconciliation: unable to send state change: udev watcher send ENI state change: eni not managed by ecs: 0a:20:e9:96:8d:92
2018-11-08T20:25:54Z [WARN] Udev watcher reconciliation: unable to send state change: udev watcher send ENI state change: eni not managed by ecs: 0a:20:e9:96:8d:92
```

**Comment by FLavalliere** [comment]
For now we simply set the error log level to ERROR

**Comment by adnxn** [comment]
> We currently provision an ASG and remove eth0 and add the static as eth1 to the VM before launching the ECS Agent.

@FLavalliere: are you using awsvpc tasks in clusters backed by instances configured like this?

**Comment by FLavalliere** [comment]
No, regular bridge mode.

**Comment by fenxiong** [comment]
```
2018-11-08T20:25:54Z [WARN] Udev watcher reconciliation: unable to send state change: udev watcher send ENI state change: eni not managed by ecs: 0a:20:e9:96:8d:92
```
@FLavalliere the value of this specific log is that, when an awsvpc task failed to launch, this log can be an indication of whether we were in the situation that the task ENI had been attached to the instance but the agent was not aware of it. The agent can't tell whether an ENI is attached by ECS but has not told the agent yet versus it's an external ENI belongs to customer. So I don't think we can stop this log in your case. Do you think dropping this log to debug level work for you?

**Comment by adnxn** [comment]
closing, released with version [v1.32.1](https://github.com/aws/amazon-ecs-agent/releases/tag/v1.32.1)
